docs/content/en/docs/getting-started/production-environment/snow-getstarted.md

```diff
@@ -0,0 +1,224 @@
+---
+title: "Create Snow production cluster" 
+linkTitle: "Snow cluster" 
+weight: 40
+description: >
+  Create a production-quality cluster on AWS Snow
+---
+
+EKS Anywhere supports an AWS Snow provider for production grade EKS Anywhere deployments.
+
+This document walks you through setting up EKS Anywhere on Snow as a standalone, self-managed cluster or combined set of management/workload clusters.
+See [Cluster topologies]({{< relref "../../concepts/cluster-topologies" >}}) for details.
+
+## Prerequisite checklist
+
+EKS Anywhere on Snow needs:
+
+* EKS Anywhere enabled Snowball devices. See [Ordering EKS anywhere enabled Snow Devices](https://docs.aws.amazon.com/snowball/latest/ug/whatissnowball.html) for ordering experience through the AWS Snow Family console.
+* To be run on an Admin instance in a Snowball Edge device. See [Initial Device Setup](https://docs.aws.amazon.com/snowball/latest/ug/whatissnowball.html) for setting up the devices, launching the Admin instance, fetching and copying the device credentials to the admin instance for `eksctl` CLI to consume.
+
+Also, see the [Ports and protocols]({{< relref "/docs/reference/ports.md" >}}) page for information on ports that need to be accessible from control plane, worker, and Admin machines.
+
+## Steps
+
+The following steps are divided into two sections:
+
+* Create an initial cluster (used as a management or standalone cluster)
+* Create zero or more workload clusters from the management cluster
+
+### Create an initial cluster
+
+Follow these steps to create an EKS Anywhere cluster that can be used either as a management cluster or as a standalone cluster (for running workloads itself).
+
+<!-- this content needs to be indented so the numbers are automatically incremented -->
+1. Set an environment variables for your cluster name
+   
+   ```bash
+   export CLUSTER_NAME=mgmt
+   ```
+
+1. Generate a cluster config file for your Snow provider
+
+   ```bash
+   eksctl anywhere generate clusterconfig $CLUSTER_NAME --provider snow > eksa-mgmt-cluster.yaml
+   ```
+
+1. Optionally import images to private registry
```

Comment:
Perhaps add a sentence here saying something like: "This optional step imports EKS Anywhere artifacts and release bundle to a local registry. This is required for air-gapped installation."